kolibri/content/fixtures/v0.4.0-beta3_content_data.json

```diff
@@ -0,0 +1 @@
,"id":"32a941fb77c2576e8f6b294cde4c3b0c"},{"available":false,"rght":11,"kind":"topic","description":"balbla3","title":"c2","level":1,"license_owner":"","parent_id":"da7ecc42e62553eebc8121242746e88a","sort_order":null,"author":"","tree_id":1,"lft":4,"license_id":2,"stemmed_metaphone":"","content_id":"1fe38e5af42f42678f8db66a18f8267a","id":"2e8bac07947855369fe2d77642dfc870"},{"available":false,"rght":6,"kind":"exercise","description":"balbla4","title":"c2c1","level":2,"license_owner":"","parent_id":"2e8bac07947855369fe2d77642dfc870","sort_order":null,"author":"","tree_id":1,"lft":5,"license_id":3,"stemmed_metaphone":"","content_id":"ce603df7c46b424b934348995e1b05fb","id":"2b6926ed22025518a8b9da91745b51d3"},{"available":false,"rght":8,"kind":"document","description":"balbla5","title":"c2c2","level":2,"license_owner":"","parent_id":"2e8bac07947855369fe2d77642dfc870","sort_order":null,"author":"","tree_id":1,"lft":7,"license_id":2,"stemmed_metaphone":"","content_id":"481e1bda1faa445d801ceb2afbd2f42f","id":"4d0c890de9b65d6880ccfa527800e0f4"},{"available":false,"rght":10,"kind":"audio","description":"balbla5","title":"c2c3","level":2,"license_owner":"","parent_id":"2e8bac07947855369fe2d77642dfc870","sort_order":null,"author":"","tree_id":1,"lft":9,"license_id":2,"stemmed_metaphone":"","content_id":"f2332710c2fd483386cdeb5ecbdda81f","id":"b391bfeec8a458f89f013cf1ca9cf33a"}],"content_file":[{"file_size":null,"checksum":"9f9438fe6b0d42dd8e913d7d04cfb2b2","supplementary":false,"contentnode_id":"32a941fb77c2576e8f6b294cde4c3b0c","extension":"mp4","lang_id":"1","preset":"high_res_video","thumbnail":false,"priority":null,"available":false,"id":"9f9438fe6b0d42dd8e913d7d04cfb2b1"},{"file_size":null,"checksum":"725257a0570044acbd59f8cf6a68b2be","supplementary":false,"contentnode_id":"32a941fb77c2576e8f6b294cde4c3b0c","extension":"mp4","lang_id":"1","preset":"low_res_video","thumbnail":false,"priority":null,"available":false,"id":"725257a0570044acbd59f8cf6a68b2bf"},{"file_size":null,"checksum":"e00699f859624e0f875ac6fe1e13d648","supplementary":false,"contentnode_id":"2b6926ed22025518a8b9da91745b51d3","extension":"","lang_id":"3","preset":"vector_video","thumbnail":false,"priority":null,"available":false,"id":"e00699f859624e0f875ac6fe1e13d649"},{"file_size":null,"checksum":"4c30dc7619f74f97ae2ccd4fffd09bf2","supplementary":false,"contentnode_id":"2b6926ed22025518a8b9da91745b51d3","extension":"mp3","lang_id":null,"preset":"low_res_video","thumbnail":false,"priority":null,"available":false,"id":"4c30dc7619f74f97ae2ccd4fffd09bf3"},{"file_size":null,"checksum":"8ad3fffedf144cba9492e16daec1e39a","supplementary":true,"contentnode_id":"2b6926ed22025518a8b9da91745b51d3","extension":"vtt","lang_id":null,"preset":"caption","thumbnail":false,"priority":null,"available":false,"id":"8ad3fffedf144cba9492e16daec1e39b"}],"content_contentnode_has_prerequisite":[{"from_contentnode_id":"32a941fb77c2576e8f6b294cde4c3b0c","to_contentnode_id":"da7ecc42e62553eebc8121242746e88a","id":1}]}
```

Comment:
Does this get loaded into the main DB, or is it just `fixtures/` a standard location for data files?

Reply:
This gets loaded into the content db for tests, I can move it, but it seemed like a consistent place to put it.